spark3/src/main/java/org/apache/iceberg/spark/FileRewriteCoordinator.java

```diff
@@ -114,6 +117,11 @@ public void commitRewrite(Table table, Set<String> fileSetIDs) {
     return newDataFiles;
   }
 
+  public void clearRewrite(Table table, String fileSetID) {
```

Comment:
Couldn't this class be used for any write that we want to hijack from Spark and commit later? It may be better to name it `FileResultSetManager` or something.